### PR TITLE
[CoffeeScript/ms-my] Added Malay (Malaysia) translation for CoffeeScript

### PR DESCRIPTION
**Pull request by hack1m**

### CODE REVIEW

File: ms-my/coffeescript-my.html.markdown [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+language: coffeescript
+contributors:
+  - ["Tenor Biel", "http://github.com/L8D"]
+  - ["Xavier Yao", "http://github.com/xavieryao"]
+filename: coffeescript.coffee
```
Comment (vendethiel) [inline ms-my/coffeescript-my.html.markdown]: please add `-ms` before `.coffee`

Comment (hack1m) [inline ms-my/coffeescript-my.html.markdown]: Done.

**Comment by vendethiel** [comment]
Thanks!

**Comment by vendethiel** [comment]
:heart_decoration:

**Pull request merged**